uber/custom_tags.py

```diff
@@ -130,6 +130,19 @@ def urlencode(s):
 
 
 @JinjaEnv.jinja_filter
+def url_to_link(url=None, text=None, target=None):
+    # Jinja2 has a 'urlize' filter but it depends on links having `.com` or `http` in the name
+    # This works with relative links and allows you to also define link text
+    if not url:
+        return ''
```

Comment:
Hm, interesting. The way I was using it, I was thinking that if there was no URL then the value would be falsey and we'd want the result to be falsey. In the bands plugin, I rely on this behavior to either display a linked "Yes" in the "W9 Uploaded" column or nothing.

That being said, if we were to use this filter to improve the admin form a bit, we'd potentially want the other behavior -- either "This attendee needs to (link)register(/link)" or "This attendee needs to register."

I'll consider it more but I think I might return the unlinked text and just manually check for the value in the bands plugin.